test/aggregate_test.exs

```diff
@@ -57,6 +57,40 @@ defmodule AshSql.AggregateTest do
     assert read_post.count_of_comments == 1
   end
 
+  describe "Context Multitenancy" do
+    alias AshPostgres.MultitenancyTest.{Org, Post, User}
+
+    test "loading a nested aggregate honors tenant" do
+      alias AshPostgres.MultitenancyTest.{Org, Post, User}
+
+      org =
+        Org
+        |> Ash.Changeset.for_create(:create, %{name: "BTTF"})
+        |> Ash.create!()
+
+      user =
+        User
+        |> Ash.Changeset.for_create(:create, %{name: "Marty", org_id: org.id})
+        |> Ash.create!()
+
+      ["Back to 1955", "Forwards to 1985", "Forward to 2015", "Back again to 1985"]
+      |> Enum.map(
+        &(Post
+          |> Ash.Changeset.for_create(:create, %{name: &1, user_id: user.id})
+          |> Ash.create!(tenant: "org_#{org.id}", load: [:last_word]))
+      )
+
+      assert Ash.load!(user, :count_visited, tenant: "org_#{org.id}")
+             |> then(& &1.count_visited) == 4
+
+      assert Ash.load!(org, :total_posts)
```

Comment:
@zachdaniel I think I was able to get it working ( sorry was in sort of a rabbit hole) .. 

1. Does this scenario supposed to be supported to start with ? It works for me without multi-tenancy
2. It for now fails cause it can't find the schema 

Am I doing something wrong or deviating for original blue print or should i try to make it work ?

Reply:
Yes, this should work as your test shows. Great work!